test/run-test.ps1

```diff
@@ -85,10 +85,10 @@ Get-ChildItem -Path $repoRoot -Recurse -Filter Dockerfile |
             if ($platform -eq "linux") {
                 $selfContainedImage = "self-contained-build-${buildImage}"
                 $optionalRestoreParams = ""
-                if ($sdkTag -like "2.0-sdk") {
-                    # Temporary workaround until 2.0 packages are released on NuGet.org
-                    $optionalRestoreParams = "-s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json -s https://api.nuget.org/v3/index.json"
-                }
+                # Enable and update as appropriate whenever pre-release packages are referenced prior to being available on NuGet.org.
+                # if ($sdkTag -like "<???>-sdk") {
+                #     $optionalRestoreParams = "-s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json -s https://api.nuget.org/v3/index.json"
+                # }
```

Comment:
I think the diffs would be cleaner if there were a `$includePrereleasePackageSourceForSdkTag = $null` (or something) conditioned on instead. That var could also be put up at the top of the file. (Although that would be more useful if there were multiple manual tweaks to gather together.)